classes/controller/Controller.php

```diff
@@ -644,7 +644,7 @@ protected function smartyOutputContent($templates)
         $this->context->cookie->write();
 
         $js_tag = 'js_def';
-        $this->context->smarty->assign($js_tag, $js_tag);
+        $this->context->smarty->assign($js_tag, \Media::getJsDef());
```

Comment:
\Media::getJsDef()) return an empty array instead of 'js_def'. Is it normal ?

Reply:
I have found with this thing, `js_def` must be an array and canno't be a string. 
This feature aim to define some JS variables directly on layout head. Also, i'm pretty sure that isn't be very used, so maybe we can simplify.

Reply:
Ok @boherm , it's ok for me to try correcting a related bug in this PR.
Maybe we should notice QA there is special check on this PR.